msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/states/ResetPasswordStates.kt

```diff
@@ -410,11 +395,6 @@ class ResetPasswordPasswordRequiredState internal constructor(
                     }
 
                     is ResetPasswordCommandResult.UserNotFound -> {
-                        Logger.warn(
```

Comment:
re. https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2374/files#r1555927999 - if we add in some form of CommandResult logging, then I agree we can remove this log here. But I would like to have a way of seeing what the CommandResult was that the internal SDK business logic produced, so that we for example can determine that it was an "unexpected result" in the SDK interface and why we're returning an Error to the developer. Wdyt?

Reply:
CommandResult has been put back in that PR.